<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiveCoinsDialog</class>
 <widget class="QDialog" name="ReceiveCoinsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>393</height>
   </rect>
  </property>
  <widget class="QFrame" name="frame2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>481</width>
     <height>150</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLineEdit" name="reqMessage">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>93</y>
      <width>288</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>An optional message that is attached to the payment request and may be displayed to the sender.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>40</y>
      <width>37</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>An optional label to associate with the new receiving address.</string>
    </property>
    <property name="text">
     <string>&amp;Label:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>reqLabel</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>93</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>An optional message to attach to the payment request, which will be displayed when the request is opened. Note: The message will not be sent with the payment over the Bitnet network.</string>
    </property>
    <property name="text">
     <string>&amp;Message:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>reqMessage</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>14</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Use this form to request payments. All fields are &lt;b&gt;optional&lt;/b&gt;.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>67</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>An optional amount to request. Leave this empty or zero to not request a specific amount.</string>
    </property>
    <property name="text">
     <string>&amp;Amount:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>reqAmount</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>120</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="reqLabel">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>36</y>
      <width>288</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>An optional label to associate with the new receiving address (used by you to identify an invoice).  It is also attached to the payment request.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>62</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Clear all fields of the form.</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="receiveButton">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>120</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>&amp;Create new receiving address</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/icons/receiving_addresses</normaloff>:/icons/receiving_addresses</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="BitnetAmountField" name="reqAmount">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>59</y>
      <width>125</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolTip">
     <string>An optional amount to request. Leave this empty or zero to not request a specific amount.</string>
    </property>
   </widget>
   <widget class="QComboBox" name="addressType">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>57</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>731</width>
     <height>161</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Requested payments history</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="recentRequestsView">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="showRequestButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Show the selected request (does the same as double clicking an entry)</string>
        </property>
        <property name="text">
         <string>Show</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/eye</normaloff>:/icons/eye</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeRequestButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Remove the selected entries from the list</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QRImageWidget" name="qr_code" native="true">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text" stdset="0">
    <string notr="true">       Click &quot;Create new receiving address&quot;</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRImageWidget</class>
   <extends>QWidget</extends>
   <header>qrimagewidget.h</header>
  </customwidget>
  <customwidget>
   <class>BitnetAmountField</class>
   <extends>QLineEdit</extends>
   <header>qt/bitnetamountfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>reqLabel</tabstop>
  <tabstop>reqAmount</tabstop>
  <tabstop>addressType</tabstop>
  <tabstop>reqMessage</tabstop>
  <tabstop>receiveButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>recentRequestsView</tabstop>
  <tabstop>showRequestButton</tabstop>
  <tabstop>removeRequestButton</tabstop>
 </tabstops>
 <resources>
  <include location="../bitnet.qrc"/>
 </resources>
 <connections/>
</ui>
